Update funnel-analysis-compare-multiple.md (#513)
amplitude/amplitude-docs: content/collections/funnel-analysis/en/funnel-analysis-compare-multiple.md

updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
updated_at: 1717015094
---

> In a Funnel Analysis chart, you can compare up to three events within a single conversion step. After [step 4 of building your funnel](/docs/analytics/charts/funnel-analysis/funnel-analysis-build), select *Compare Event* from the *Options* fly-out menu, and then select the events to compare.

Reviewer: ```suggestion
In a Funnel Analysis chart, you can compare up to three events within a single conversion step. After [step 4 of building your funnel](/docs/analytics/charts/funnel-analysis/funnel-analysis-build), select *Compare Event* from the *Options* flyout menu, and then select the events to compare.
```

![multiple_CEsin_funnel.png](/docs/output/img/funnel-analysis/multiple-cesin-funnel-png.png)

> Comparing multiple events within one step is available for conversion and conversion over time metrics. Three events may be compared per step, for up to two steps at a time. Event comparisons are available for all three order options (any order, this order, exact order); however, they are not available in dashboard filters, and will be removed if a user switches between charts or funnel metrics.

Reviewer: ```suggestion
Comparing multiple events within one step is available for conversion and conversion over time metrics. Compare up to three events per step, and two steps at a time. Event comparisons are available for all three order options (any order, this order, exact order); however, they aren't available in dashboard filters, and are removed if you switch between charts or funnel metrics.
```
